services/product-service/src/functions/createProduct/index.ts

```diff
@@ -0,0 +1,23 @@
+import { handlerPath } from "@libs/handler-resolver";
+import schema from "./schema";
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      http: {
```

Comment:
you need an additional property to generate request body in swagger:
"bodyType: 'CreateProductRequestBody'" where the Create...Body defined as interface/type